tests/integration/plugins/python_v2/test_python/requirements.txt

```
@@ -0,0 +1 @@
+Flask
```

```suggestion
Flask>=2.32.0
```

Should have equivalent behaviour but will silence the OSV scanner.

Alternatively (and perhaps preferably) we can exclude these test requirements files from the scans.